app/controllers/admin/npq/applications/edge_cases_controller.rb

```diff
@@ -13,9 +13,13 @@ class EdgeCasesController < Admin::BaseController
 
         def index
           query_string = params[:query]
+          funding_eligiblity_status_code = params[:FundingEligiblityStatusCode]
+          employment_type = params[:EmploymentType]
+          start_date = convert_date(params["start_date(3i)"], params["start_date(2i)"], params["start_date(1i)"])
```

Comment:
is there anyway to give more human readable param names? something like `start_date(day)`, `start_date(month)` etc

Reply:
Yeah that's fiddly. Left a comment for now, Went through the [documentation for this component here ](https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder/Builder#govuk_date_field-instance_method)and couldn't find anything out of the box that we can use.